db/models.js

```diff
@@ -79,12 +84,12 @@ const deleteById = (id, callback) => {
     });
 };
 
-const addAuthorAndBook = (author, book, callback) => {
-  const authorQuery = `INSERT INTO authors (name, followers, biography, author_image, createdAt, updatedAt) VALUES ('${author.name}', '${author.followers}', '${author.biography}', '${author.author_image}', '${author.createdAt}', '${author.updatedAt}')`;
+const addAuthorAndBook = (author, book, callback) => { // update to reflect new id column
+  const authorQuery = `INSERT INTO authors (name, followers, biography, author_image) VALUES ('${author.name}', '${author.followers}', '${author.biography}', '${author.author_image}')`;
```

Comment:
If you're using `sequelize` already, then I would use whatever functions/utilities it provides to interact with the database.